README.md

```diff
@@ -1,13 +1,11 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
+This API was built with Node.js, Mongoose, MongoDB and hosted by Google Cloud and MongoDB Atlas
+It is the backend for a previous project "Happy Thoughts", frontend here:  
+https://github.com/joeohm/project-happy-thoughts
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+API: https://project-happy-thoughts-api-rwwjpm7rkq-uc.a.run.app  
+Frontend: https://joels-happy-thoughts.netlify.app/
```

Comment:
Good idea to have both the backend and frontend in the readMe so you can see both "ends" ;)